### PR TITLE
Adds docker-related utils

### PR DESCRIPTION
**Pull request by claudiubelu**
This function runs the given command in the given docker image.

### CODE REVIEW

File: k8s_test_harness/util/docker_util.py [hunk 1]
```diff
@@ -12,3 +12,39 @@ def ensure_image_contains_paths(image, paths):
             ["docker", "run", "--rm", image, "ls", "-l", path],
             check=True,
         )
+
+
+def run_in_docker(image: str, check_exit_code: bool, *command: str):
```
Comment (bschimke95) [inline k8s_test_harness/util/docker_util.py]: Nit: you could add kwargs for extra docker flags

**Review by bschimke95 — APPROVED** [review]
LGTM

File: k8s_test_harness/util/docker_util.py (hunk 1 above)
Comment (claudiubelu) [reply k8s_test_harness/util/docker_util.py]: Added ``docker_args``, which can also cover single argument flags, not just pairs.

**Comment by bschimke95** [comment]
linter issues are addressed in #19

**Pull request merged**